docs/swagger.yml

```diff
@@ -177,13 +194,49 @@ paths:
             $ref: "#/definitions/CredentialOut"
         400:
           description: "Invalid input"
+        401:
+          description: "Not authorized"
+        404:
+          description: "Credential not found"
+    patch:
+      tags:
+        - "Credential"
+      summary: "Partial update of an existing credential"
+      description: "Partial update an existing credential."
```

Comment:
Same ? as above. 'Partially update' or 'Partial update of an' instead of 'Partial update an'.

Reply:
Fixed